test/common.c

```diff
@@ -217,6 +217,7 @@ gboolean test_make_filesystem(const gchar *dirname, const gchar *filename)
 			&error,
 			"/sbin/mkfs.ext4",
 			"-F",
+			"-I256",
```

Comment:
This is unrelated and maybe of increasing significance for the users of RAUC. Maybe it would be worth making a separate PR for documentation purposes?

Reply:
Yes, will drop it from this PR.